sunspec/core/util.py

```diff
@@ -95,6 +96,12 @@ def data_to_double(data):
         return d[0]
 
 def data_to_str(data):
+
+    # Change the data from bytes string to regular string for python 3
+    # compatibility
+    if sys.version_info > (3,):
+        data = str(data, 'latin-1')
```

Comment:
Is `latin-1` the encoding required by the SunSpec standard?  If not, we probably don't want to reference it.  Also, shouldn't encoding/decoding happen even in py2?  Sure, bytes and str are the same in py2 but if there actually is anything to encode or decode then it will still have an effect.  If SunSpec restricts to ascii then we should here as well to get an exception for anything invalid.